lib/matplotlib/axes/_base.py

```diff
@@ -487,14 +487,14 @@ def __init__(self, fig, rect,
             self._shared_x_axes.join(self, sharex)
             if sharex._adjustable == 'box':
                 sharex._adjustable = 'datalim'
-                #warnings.warn(
+                # warnings.warn(
                 #    'shared axes: "adjustable" is being changed to "datalim"')
```

Comment:
I just fixed for it to conform with PEP8 but the code is out of scope of this PR.
Maybe make a new PR (and I should probably remove the whitespace PEP8 fix)? @fariza ?